aip/general/0203.md

```diff
@@ -171,6 +163,15 @@ A resource with an unordered list **may** return the list in a stable order, or
 
 ## Rationale
 
+### Required set of annotations
+
+A field used in a request message must be either an input or an output.
+
+In the case of an output, the `OUTPUT_ONLY` annotation is sufficient.
+
+In the case of an input, a field is either required or optional, and therefore
```

Comment:
We specifically didn't want to mandate OPTIONAL. (There was debate on whether to even provide it.)

Reply:
Yes, and after a decent amount of discussion it was agreed that field_behavior should be required: https://github.com/aip-dev/google.aip.dev/pull/1100. See the rationale section for more details, but this choice has lead to a lot of incorrectly documented fields since developers did not think through the field behavior choice. field behavior is being used as forcing function.